composer.json

```
@@ -3,7 +3,7 @@
     "minimum-stability": "stable",
     "prefer-stable": true,
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```

Ah bah non, il fallait plutôt écrire ça :

```suggestion
        "php": ">=8.2.0",
```